polyfill/test/parseResults.js

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env node
+
+const { stdin, stdout, exit } = process;
+let testOutput = '';
+const now = Date.now();
+
+stdin.setEncoding('utf8');
+
+stdout.write('Starting Test262 tests.\n');
+
+let testCount = -1;
+let lastReportedCount = -1;
+
+function reportStatus() {
+  const secs = (Date.now() - now) / 1000;
+  stdout.write(`\r${testCount - 1} tests completed in ${secs} seconds. (${Math.round(testCount / secs)} tests/sec)   `);
```

Comment:
For the final count, let's use `tests.length` from the 'end' handler.

Reply:
good idea. done!